### ISSUE TITLE
about efficientnet-b7  train on cifar-10,but i set up batch size is 8 ,gpu(1080ti 11g) out of memory 

### ISSUE THREAD

**Issue by daixiangzi** [lukemelas/EfficientNet-PyTorch#49]
i guess if the efficientnet-b7 model size is large?

**Comment by WonderSeven** [comment]
Yes, the efficientnet-b7 is too large to load on one 1080ti GPU.

**Comment by ksenobojca** [comment]
I believe actual blocker for training/tuning is that with current implementation we cannot use torch.utils.checkpoint.checkpoint_sequential - PyTorch way to reduce memory during training. Some minor refactor should do the trick. Possible guide: https://github.com/prigoyal/pytorch_memonger
